tests/testthat/test-tibble.R

```diff
@@ -6,6 +6,7 @@ test_that("use_tibble() requires a package", {
 test_that("use_tibble() Imports tibble", {
   create_local_package(path_temp("mypackage"))
   withr::local_options(list(usethis.quiet = FALSE))
+  ui_silence(use_package_doc())
   expect_snapshot(use_tibble())
```

Comment:
The `use_tibble()` test should probably gain the `with_mock(check_installed = function(pkg) TRUE`, ...)` treatment, since you're here @malcolmbarrett .

Mostly, for consistency. But also because my recent-ish re-removal of rematch2 ?I think? means tibble is no longer an indirect dependency. tibble is still present for testing, just because styler (suggested) depends on it, I think. So it's sort of coincidence that this test is currently passing. https://github.com/r-lib/usethis/issues/1223

Reply:
Fair enough. Updated!